store/migration/dev/LATEST.sql

```diff
@@ -659,7 +659,8 @@ CREATE TABLE task (
     status TEXT NOT NULL CHECK (status IN ('PENDING', 'PENDING_APPROVAL', 'RUNNING', 'DONE', 'FAILED', 'CANCELED')),
     type TEXT NOT NULL CHECK (type LIKE 'bb.task.%'),
     payload JSONB NOT NULL DEFAULT '{}',
-    earliest_allowed_ts BIGINT NOT NULL DEFAULT 0
+    earliest_allowed_ts BIGINT NOT NULL DEFAULT 0,
+    rollback_by INTEGER NOT NULL DEFAULT 0
```

Comment:
not sure if we should have rollback_from instead.

What if we create multiple rollback tasks for the same task (and only process one of them)?

Reply:
SG. Two concurrent requests may create two identical rollback issues for the same task. They both have the same rollback_from field, and the user needs to close one.